### DISCUSSION TITLE
Jetbot will no longer boot.

### DISCUSSION THREAD

**Discussion by tjp1940** [NVIDIA-AI-IOT/jetbot#405] [Q&A]
Hello.  I built a jetbot and grealty enjoyed running the jupyter notebooks on basic_motion and teleoperation.  The whole concept is brilliant.   While working the notebooks I did experiment somewhat.  The next time I tried to run a notebook my jetbot would not respond.  I spent a week so far reading  the developers forum and reading documentation.  It was suggested that I use a serial connection with my nano and a computer, which I did.  I got an error message on my monitor that ended with:
                         ...............
       "[0000.353] Nano-SD: checking PT table on QSPI ...
        [0000.358] PT: Partition TBC NOT found !
        [0000.361] Warning: Find Partition via PT Failed
        [0000.366] Error is 1"

And then lost control of the serial connection and could not issue any more commands.   From what I see I figure that there is a corrupt boot loader and the  "PT:Partition TBC"  needs to be restored.  So I went to the documentation again looking for a way to re-flash the nano board.  Well that was like going into a snake-infested jungle butt-naked.  They tell you how to build all of their nvidia devices with great detail - all in the same  document too.  The documentation is excellent for a knowledgeable software developer,  but for a novice like myself with limited experience, it is incomprehensible .

Finally, to the point.  Could someone please tell me if there is a source where it leads you through the steps necessary to re- flash the jetson nano 4gb SDK B01 and includes the necessary software files ?  
Thank you,

**Comment by jaybdub** [comment]
Hi @tjp1940,

Thanks for reaching out!

Apologies for the frustration you're facing here.  When you tried to flash, did you use the JetBot image directly, or the original JetPack 4.5 SD card image?  (or something else?)

https://developer.nvidia.com/embedded/jetpack

I believe there were changes to the bootloader between JetPack 4.4.1 / JetPack 4.5, so perhaps this is related.  From my experience, I had no issues transitioning, but perhaps there was something different with our environments.

Please let me know if you have any questions.

Best,
John

**Comment by tjp1940** [comment]
John, I never got that far.  I was going to try but then realized I could
not make a connection.  I do not think the nano booted enough to
communicate with putty.  Or then perhaps I need some other putty setup.  I
followed JetsonHacksNano video.
TP

**Reply by jaybdub** [reply]
If possible, do you have a monitor, keyboard that you could connect just to test that the simple setup is working as expected?

Best,
John

**Comment by tjp1940** [comment]
Yes. I will load JetPack 4.3 SD, connect the monitor,keyboard,
direct connection to internet , and power in a minute.

**Comment by tjp1940** [comment]
Just completed test.  The nano shows no signs of life.  No boot up, no
NVIDIA logo or boot up text , just a  'NO IMAGE/NO SIGNAL ' message from
the terminal and then the black screen. The error message I mentioned
earlier cam via the USB TTY cable using Putty.
TP

**Comment by santaimpersonator** [comment]
Try the plain jetpack 4.5 image, as noted in the call out at the bottom of **Step1** here: https://jetbot.org/master/software_setup/sd_card.html

It'll flash the updated bootloader, but you'll need to connect a monitor (keyboard and mouse) to run through the system's oem-config steps. If that doesn't work @jaybdub will probably be your best source for help.
